framework/validators/UniqueValidator.php

```diff
@@ -194,6 +194,15 @@ private function modelExists($targetClass, $conditions, $model)
                 
                 // any with relation can't be loaded because related fields are not selected
                 $query->with = null;
+    
+                if (is_array($query->joinWith)) {
+                    // any joinWiths need to have eagerLoading turned off to prevent related fields being loaded
+                    foreach ($query->joinWith as &$joinWith) {
```

Comment:
Would setting `$query->joinWith` to `null` work?

Reply:
Structurally yes, my original push did just that.

However, I think that would produce unexpected results if one was added since `joinWith` modifies the original `find()` query.